ladybug/psychrometrics.py

```diff
@@ -29,76 +28,58 @@ def find_saturated_vapor_pressure_high_accuracy(t_kelvin):
         crit_temp = 647.096 / t_kelvin
         exponent = crit_temp * express_result
         power = math.exp(exponent)
-        saturation_pressure = power * 22064000
+        p_ws = power * 22064000
     else:
         # Calculate saturation vapor pressure below freezing
         theta = t_kelvin / 273.16
         exponent2 = ((1 - (theta**(-1.5))) * (-13.928169)) + \
             ((1 - (theta**(-1.25))) * 34.707823)
         power = math.exp(exponent2)
-        saturation_pressure = power * 611.657
+        p_ws = power * 611.657
+    return p_ws
 
-    return saturation_pressure
 
-
-def find_humid_ratio(air_temp, rel_humid, bar_press=101325):
-    """
-    Calculates Humidity Ratio (kg water/kg air), Partial Pressure (Pa), and
-    saturation_pressure (Pa) at a given
-    at Temperature air_temp (C), Relative Humidity rel_humid (%),
-    and Barometric Pressure bar_press (Pa).
+def humid_ratio_from_db_rh(db_temp, rel_humid, b_press=101325):
+    """Humidity Ratio (kg water/kg air) at a db_temp (C),
+    rel_humid (%) and Pressure b_press (Pa).
     """
     # Find saturation pressure
-    t_kelvin = air_temp + 273
-    saturation_pressure = find_saturated_vapor_pressure_high_accuracy(t_kelvin)
+    p_ws = saturated_vapor_pressure(db_temp + 273)
     # Calculate partial pressure
     decrh = rel_humid * 0.01
-    partial_pressure = decrh * saturation_pressure
+    p_w = decrh * p_ws
     # Calculate humidity ratio
-    press_differ = bar_press - partial_pressure
-    constant = partial_pressure * 0.621991
+    press_differ = b_press - p_w
+    constant = p_w * 0.621991
     humidity_ratio = constant / press_differ
+    return humidity_ratio
 
-    return humidity_ratio, partial_pressure, saturation_pressure
 
-
-def find_enthalpy(air_temp, humid_ratio):
-    """
-    Calculates Enthalpy (kJ/kg) at Humidity Ratio humid_ratio (kg water/kg air)
-    and at Temperature air_temp (C).
+def enthalpy_from_db_hr(db_temp, humid_ratio):
+    """Enthalpy (kJ/kg) at humid_ratio (kg water/kg air) and at db_temp (C).
     """
-    en_variable1 = 1.01 + (1.89 * humid_ratio)
-    en_variable2 = en_variable1 * air_temp
-    en_variable3 = 2500 * humid_ratio
-    en_variable4 = en_variable2 + en_variable3
-    if en_variable4 >= 0:
-        enthalpy = en_variable4
-    else:
-        enthalpy = 0
-
-    return enthalpy
+    enthalpy = ((1.01 + (1.89 * humid_ratio)) * db_temp) + (2500 * humid_ratio)
+    return enthalpy if enthalpy >= 0 else 0
 
 
-def find_wet_bulb(db_temp, rh, psta=101325):
-    """
-    Calculates Wet Bulb Temperature (C) at Temperature db_temp (C),
-    Relative Humidity rh (%), and Barometric Pressure psta (Pa).
+def wet_bulb_from_db_rh(db_temp, rh, b_press=101325):
```

Comment:
I'm in favor of name changes and removing *find* or *calculate* form method names but do we really need *from_x_y* after the names? If there are 5 different methods for wetbulb calculation then we probably want to differentiate them by add *from_x_y* similar to how dew_point methods are but in this case and other cases like enthalpy I'm not sure if we need those.